bazel/linux/defs.bzl

```diff
@@ -109,7 +113,13 @@ def _kernel_modules(ctx):
         # Force GCC to produce colored output
         cflags += " -fdiagnostics-color=always"
 
-        extra.append("EXTRA_CFLAGS='%s'" % cflags)
+        for include in depset(includes).to_list():
+            cflags += " -I $PWD/%s" % include
+
+        for include in depset(quote_includes).to_list():
+            cflags += " -iquote $PWD/%s" % include
+
+        extra.append("EXTRA_CFLAGS=\"%s\"" % cflags)
```

Comment:
minor nit: can you change the `=` to `+=`? I believe it's supported by `make`, I'm worried we'll need to pass `EXTRA_CFLAGS=...` via `extra` attribute manually, and won't notice it does not work because it is overridden here.

Reply:
Good point. Fixed.